service/kvm/v1/resource/kvmclusterconfig/create_test.go

```diff
@@ -0,0 +1 @@
+package kvmclusterconfig
```

Comment:
Ok. I will refactor this. Is there a proper documentation / tutorial / getting started guide / wiki page for `operatorkit` anywhere? I know there's Godoc, but it isn't immediately obvious in communicating the core idea behind the framework and how it should be used.

Reply:
AFAIK there is no documentation that helps you understanding the basic concept I guess. We should maybe just create something. Since you are in the situation of missing certain information, we can maybe put some bullet points together, just to have something as starter.

Reply:
I'll make some notes during my progress here to get some start for the guide.

Reply:
To give you a hint. Say I see path: `service/awsconfig/v1/resource/cloudformation/*.go` - I know this resource reconciles cloudformation templates based on AWSConfig object being watched.